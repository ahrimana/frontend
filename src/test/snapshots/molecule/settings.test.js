/* eslint no-undef: 0 */
/* eslint no-unused-expressions: 0 */
/* eslint no-sequences: 0 */
/* eslint quotes: 0 */

import React from 'react';
import Settings from '../../../components/molecules/settings';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("Service Molecule (snapshot)", () => {
  const setup = () => shallow(<Settings />, { router: {} });

  it("Settings snapshot", () => {
    const component = setup();
    expect(toJson(component)).toMatchSnapshot();
  });
});
