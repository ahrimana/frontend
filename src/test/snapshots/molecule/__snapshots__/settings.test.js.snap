exports[`Service Molecule (snapshot) Settings snapshot 1`] = `
<MenuItem
  checked={false}
  desktop={false}
  disabled={false}
  focusState="none"
  insetChildren={false}
  leftIcon={<ActionInput />}
  onTouchTap={[Function]}
  primaryText="Logout"
  style={
    Object {
      "cursor": "pointer",
    }
  } />
`;
